app/src/components/multistepform.js

```diff
@@ -0,0 +1,63 @@
+import {Children, useState} from 'react';
+
+export default function MultiStepForm({children, submitText, handleSubmit}) {
+
+  const [step, setStep] = useState(0);
+
+  function handleNextStep(e) {
+    setStep(step+1);
+  }
+
+  function handleBackStep(e) {
+    setStep(step-1);
+  }
+
+  return (
+    <div className="flex h-screen rounded-lg border border-gray-300 h-fit min-h-96 relative p-4">
```

Comment:
There's a lot of long classNames in this file. Is it possible to extract these to a css file or something? I have no idea how tailwind works so I'm just wondering

Reply:
Yeah this is a tailwind thing. You're supposed to use their classes instead of direct css. The tailwind classes are designed to handle responsive behavior by default. Whereas with straight css you'd need to create manual media queries.

Reply:
Hmm okay, that's something I'll have to learn to deal with then. Thanks for the clarification